engine/apps/api/tests/test_alert_receive_channel.py

```diff
@@ -852,35 +852,6 @@ def test_update_alert_receive_channels_direct_paging(
     assert response.json()["detail"] == AlertReceiveChannel.DuplicateDirectPagingError.DETAIL
 
 
-@pytest.mark.django_db
-def test_delete_alert_receive_channel_direct_paging_duplicate(
-    make_organization_and_user_with_plugin_token, make_team, make_alert_receive_channel, make_user_auth_headers
-):
-    """Check that it's possible to delete direct paging integration even if there is a duplicate for the team."""
```

Comment:
Duplicate direct paging integrations are now impossible, so removing this test

Reply:
Maybe add some test which raises a DB error when trying to create a duplicate entry?